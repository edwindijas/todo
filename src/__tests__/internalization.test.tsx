import { render } from "@testing-library/react";
import Internalization from "@/Internalization";

it('internalization matches snapshot', () => {
  const {container} = render(<Internalization ><div >Hello World</div></Internalization>)
  expect(container).toMatchSnapshot()
} )