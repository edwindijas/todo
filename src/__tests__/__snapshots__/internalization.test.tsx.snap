// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`internalization matches snapshot 1`] = `
<div>
  <div>
    Hello World
  </div>
</div>
`;
